src/mattack_actors.cpp

```diff
@@ -691,14 +683,24 @@ bool wife_u_actor::call( monster &z ) const
     target->add_effect( effect_lust, 8_turns + 1_turns * rng( 1, fake_dex ) );
     z.add_effect( effect_lust, 8_turns + 1_turns * rng( 1, target->dex_cur ) );
 
-    if( has_cum( target ) ) {
+    if( target->get_effect_int( effect_lust ) >= 100 ) {
+        target->add_msg_player_or_npc( m_info,
+                                       _( "You go to heaven!" ),
+                                       _( "<npcname> goes to heaven!" ) );
+        target->remove_effect( effect_lust );
+        target->mod_moves( -50 );
         g->m.add_item( target->pos(), item( "h_semen", calendar::turn ) );
```

Comment:
今更ですが、よろしくない単語が目につきました！
グーグル先生によると、「bodily fluid」という単語があるそうです
正しい意味は、リンパとか血漿の総称らしいです

```suggestion
        g->m.add_item( target->pos(), item( "h_bodily fluid", calendar::turn ) );
```

Reply:
確かにこの単語もあまり宜しくないですね。
「bodily fluid」というのは良い感じにぼかせてる気がするので採用させていただきます。